cou/cli.py

```diff
@@ -103,7 +102,7 @@ async def analyze_and_plan(
     progress_indicator.start(f"Connecting to '{model_name or 'current-model'}' model...")
```

Comment:
When connecting to the current model, the message will be "Connecting to 'current-model' model...", which looks a bit weird and gives the impression that 'current-model' is an actual name of a model. Could you please rephase it?

Reply:
It's known naming from Juju, that's why I think it's ok. Do you have any suggestions how to re-phase it?

Reply:
@rgildein How hard it is to access `/home/<USER>/.local/share/juju/model.yaml` and access `current-model`?

Reply:
It should not be hard, but I do not think this is related with PR. Maybe I should reworked it back at it was (not functional indicator), since this PR is focused to improve KeyboardInterrupt.

Reply:
I reverted changes, so we can focus on goal of this PR.

Reply:
Resolving this conversation, since I created #171.